src/widgets/study-path/study-path.test.tsx

```diff
@@ -23,7 +24,7 @@ vi.mock('@/shared/hooks/use-data-by-name', () => ({ useDataByName: vi.fn().mockI
 
 describe('StudyPath Component', () => {
   it('renders the title and paragraph text correctly for angularPath', () => {
-    render(<StudyPath path="angular" />);
+    render(<MemoryRouter><StudyPath path="angular" /></MemoryRouter>);
```

Comment:
nit: for such cases we have the `renderWithRouter` wrapper

Reply:
thank you for information, changed.